src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ *  in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related
+ * CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arraySourceString = sourceString.split(';');
+
+  return arraySourceString.reduce((stylesObject, styles) => {
+    if (!styles.length) {
+      return stylesObject;
+    }
+
+    const [key, value] = styles.split(':').map((item) => item.trim());
+
+    if (!value) {
```

Comment:
You can try use here ternary operator instead of if:

```suggestion
    return !value
  ? stylesObject
  : {
      ...stylesObject,
      [key]: value,
    };
```